monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -1,43 +1,43 @@
+Ocultar a los remitentes en Monero es algo más complicado que solo proteger direcciones. Todo Monero está representado en la cadena de bloques como salidas de transacciones, llamadas TXOs. Cada TXO es etiquetado usando un ID único (una sola vez) llamado clave pública (también es llamado una dirección secreta). Con sólo ver en la cadena de bloques de Monero, las claves públicas de una TXO absolutamente no se pueden conectar a las direcciones de Monero. Sin embargo, las claves públicas por si mismas también necesitan protección. Sería inaceptable que el flujo de Monero entre las claves públicas fuera transparente, considerando que las técnicas fuera de la cadena de bloques podrían conectar esas claves públicas con las personas.
 
-Monero movement among public keys is kept secret right now using what are called cryptographic ring signatures. With this approach, the public key for an actual TXO is hidden by grouping it with decoys (also called mixins). Looking at a transaction, there is no way to tell if the real TXO is being used or one of the decoys. You can see example input and output TXOs by looking up a transaction in a Monero blockchain explorer, such as [moneroblocks.info](https://moneroblocks.info/) or [xmrchain.net](https://xmrchain.net/).
+El movimiento de Monero entre claves públicas se mantiene en secreto en este momento utilizando lo que se llaman firmas de círculo criptográfico. Con este enfoque, la clave pública para un TXO real se oculta agrupándola con señuelos (también llamados mixins). Al mirar una transacción, no hay manera de saber si se esta utilizando el TXO real o uno de los señuelos. Puede ver ejemplos de entrada y salida de TXOs en un explorador de la cadena de bloques de Monero, como [moneroblocks.info](https://moneroblocks.info/) o [xmrchain.net](https://xmrchain.net/).
 
-How TXO decoys are selected from the blockchain has improved over the years. Initially, as a built-in part of the protocol, decoys were optional. Over time the number of decoys was increased and mandated to give the requirement of exactly 10 decoys today. Also, initially in the standard wallet, decoys were randomly sampled uniformly over all transactions. The sampling process improved over time to give the tailored random process, based on picking times from a special gamma distribution, that we have today.
+La forma en que se seleccionan los señuelos de un TXO de la cadena de bloques ha mejorado con los años. Inicialmente, como parte incorporada del protocolo, los señuelos eran opcionales. Con el tiempo, el número de señuelos se incrementó y en la actualidad se ha ordenado para dar el requisito de que sean exactamente 10 señuelos. Además, inicialmente en el monedero estándar, los señuelos fueron muestreados al azar uniformemente sobre todas las transacciones. El proceso de muestreo mejoró con el tiempo para dar el proceso aleatorio a medida, basado en los tiempos de selección de una distribución gamma especial, que tenemos hoy en día.
```

Comment:
```suggestion
La forma en que se seleccionan los señuelos de un TXO desde la cadena de bloques ha mejorado con los años. Inicialmente, como parte incorporada del protocolo, los señuelos eran opcionales. Con el tiempo, el número de señuelos se incrementó y dar hoy exactamente 10 señuelos en el requerimiento. Además, inicialmente en el monedero estándar, los señuelos fueron muestreados al azar de manera uniforme sobre todas las transacciones. El proceso de muestreo mejoró con el tiempo, basado en la selección de tiempos desde una distribución gamma especial, para dar el proceso aleatorio a medida que tenemos hoy.
```

Reply:
This is my suggestion:

"La forma en que se seleccionan los señuelos de un TXO **desde** la cadena de bloques ha mejorado con los años. Inicialmente, como parte incorporada del protocolo, los señuelos eran opcionales. Con el tiempo, el número de señuelos se incrementó y **dar hoy exactamente 10 señuelos en el requerimiento**. Además, inicialmente en el monedero estándar, los señuelos fueron muestreados al azar **de manera uniforme** sobre todas las transacciones. El proceso de muestreo mejoró con el tiempo, **basado en la selección de tiempos desde una distribución gamma especial, para dar el proceso aleatorio a medida que tenemos hoy.**"

Reply:
Con el tiempo, el número de señuelos se incrementó y **dar hoy en el requerimiento exactamente 10 señuelos**...  I think this sentence in Spanish is not fully understood.

basado en la selección de tiempos desde una distribución gamma especial, para dar el proceso aleatorio a medida que tenemos hoy." Done

Reply:
I have other options:
Original text:
"Over time the number of decoys was increased and mandated to give the requirement of exactly 10 decoys today."

Suggestions:
- "Con el tiempo, el número de señuelos se incrementó y **al día de hoy se decidio por requerimiento que diera exactamente 10 señuelos.**"
- "Con el tiempo, el número de señuelos se incrementó y **en la actualidad, por decisión en requerimiento dar exactamente 10 señuelos.**"

_Literal translation_
- "Con el tiempo, el número de señuelos se incrementó y **al día de hoy, por mandato dar el requerimienro exacto de 10 señuelos.**"
- "Con el tiempo, el número de señuelos se incrementó y **en la actualidad, por mandato dar el requerimienro exacto de 10 señuelos.**"

The word, "orden", is a direct order from someone. I could say, that by order adding 10 decoys were added to each transaction, but it was actually not an order, it was a decision, a mandate, decided by the Monero Devs to add the decoys. In the end, it is an order. But I would keep mandate literal translation, "mandato", or "decisión", as for a decision taken by the Monero Devs to increase the decoys as a requirement directly from the creation of each transaction.

Reply:
Yes, I agree with the second sentence

"Con el tiempo, el número de señuelos se incrementó y al día de hoy se decidio por requerimiento que diera exactamente 10 señuelos."